src/containers/app/AppActionFactories.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * @flow
+ */
+
+import * as actionTypes from './AppActionTypes';
```

Comment:
A few notes on convention I've been following:

1. Please rename to `ActionFactory`. We're using the term `Factory` pretty loosely here, but nevertheless, it's a single Factory.
2. I've started combining `ActionTypes` and `ActionFactory` files into just a single `ActionFactory` file.

Reply:
yeah "factory" makes more sense